templates/glue-spark-job.yaml

```diff
@@ -111,11 +111,15 @@ Resources:
         Name: !Ref JobCommand
         ScriptLocation: !Ref S3ScriptLocation
       DefaultArguments:
-        --TempDir: !Sub s3://${S3OutputBucketName}/tmp
+        --TempDir: !Sub s3://${S3BucketName}/tmp
         --enable-continuous-cloudwatch-log: !Ref ContinuousLog
+        --enable-metrics: true
+        --enable-spark-ui: true
+        --spark-event-logs-path: !Sub s3://${S3BucketName}/spark-logs/${AWS::StackName}/
```

Comment:
Lines 117 and 118 were the main reason for this PR

Reply:
Are we writing the logs to the same bucket as our JSON datasets?

Reply:
Yes, to the "intermediate" bucket, which is basically the work bucket. Do you think that's problematic?

Reply:
No, that makes sense.